Resources/Documentation/dv_info.md

```diff
@@ -1,32 +1,126 @@
 
-## Bitstream Error Concealment
+7. When using dvcapture in the vrecord GUI, environment parameters can be adjusted in the "Config" tab.
+
+![dv_vrecord_configuration2](../dv_vrecord_configuration2.jpg)
+
+
+
+## Playback
+
+![dv_vrecord_playback](../dv_vrecord_playback.jpg)
+
+1. Select the name of the DV device you want to use from the "Select a DV Device" list. If you have two decks of the same make and model connected, please see the "DV Settings" section for steps to determine the device identifier for each one.
+
+2.  Click the "Deck Control" button to allow vrecord to control your device. The status should switch to display "stopped" in the status field (the status will say "disabled" if "Deck Control" is not selected). 
+
+3.  Use the corresponding buttons to fast forward, rewind, play and stop the tape. These buttons will not respond if "Deck Control" is disabled. The status of the deck (i.e. stopped, rewinding, play, etc.) will be displayed in the "Status" field based on what the deck is currently doing.
+
+4. Click the "Repack" to repack your tape (fast forward the tape all the way and then rewind it). This button will not respond if "Deck Control" is disabled. The status of the deck (i.e. rewinding or fast forwarding) will be displayed in the "Status" field as the tape is repacked.
+
+5. To preview the tape, click on the "Playback" button in the bottom right of the "Settings" window. The settings menu will close and the Terminal window will display the settings you have entered.
+
+6. A mpv window will open displaying the playback of the tape. It may take ~20 seconds for the deck to respond and begin playing.
+> **_NOTE:_** If the timecode does not start at the very beginning of the tape, the playback window will not open until the timecode is detected. As soon as the counter starts moving on your deck and in the Terminal window, the viewer window should pop-up and begin playback. ![dv_vrecord_timecode-record-start](../dv_vrecord_timecode-record-start.gif)
+
+7. To end playback, click the `x` in the upper left hand corner of the playback window or hit the "esc" key.
+
+
+## Recording
+
+![dv_vrecord_record-instructions](../dv_vrecord_record-instructions.mp4)
+
+1. Select the name of the DV device you want to use from the "Select a DV Device" list.
+
+2. Click the "deck control" button to allow vrecord to control your device.
+
+3. To record, select the recording directory (location where you want to save the file) and enter the recording name, name of the person digitizing this tape, recording time in minutes (if applicable) and any other options you would like applied to your file.
+
+4. Click the "Record" button. The settings menu will close and the Terminal window will display the settings you have entered.
 
+5. When you are ready hit the "Return" button to begin the recording.
+
+6. A mpv window will open displaying the playback of the tape once dvrescue starts the recording. 
```

Comment:
Drop `mpv` (not actually mpv and don't really need to explain what is running the window)

Reply:
my bad!